extensions/cornerstone/src/utils/measurementServiceMappings/Bidirectional.js

```diff
@@ -67,6 +67,7 @@ const Bidirectional = {
       metadata,
       referenceSeriesUID: SeriesInstanceUID,
       referenceStudyUID: StudyInstanceUID,
+      frameNumber: mappedAnnotations[0].frameNumber || 1,
```

Comment:
same

Reply:
doen in fed74aae7ada884c3563b114ed8bc46938e325b3